ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -301,6 +301,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8
+- as: e2e-aws-ipi-sdn-migration-ovn
```

Comment:
Follow the rules in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md

Reply:
`e2e-aws-ipi-sdn-migration-ovn-p2` is ok?